Control/InvertedPendulumLQRControl/inverted_pendulum_lqr_control.py

```diff
@@ -0,0 +1,186 @@
+"""
+Inverted Pendulum LQR control
+author: Trung Kien - letrungkien.k53.hut@gmail.com
+"""
+
+import math
+import time
+
+import scipy.linalg as la
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Model parameters
+
+l_bar = 2.0  # length of bar
+M = 1.0  # [kg]
+m = 0.3  # [kg]
+g = 9.8  # [m/s^2]
+
+Q = np.diag([0.0, 1.0, 1.0, 0.0])
```

Comment:
Could you add comments for Q and R? What is the Q and R for?

Reply:
Thank @AtsushiSakai, I will fix it soon. Can I create folder inverted_pendulum and move implement of MPC and LQR to it (something like move_to_pose). I think it will be easier to follow, maybe I will add more controllers like PID, pole placement ... to this folder.
Thanks.